src/DetailView/DetailView.js

```diff
@@ -22,15 +23,41 @@ class DetailView extends ConnectionComponent {
 			filteredEvents: [],
 			currentEventType: null,
 			filter: [],
+			eventChunkLoading: false,
 		};
 		this.includeEventChildren = false;
 		this.events = [];
+		this.eventChunk = [0, config.eventTableChunkSize];
 
 		this.handleEventTypeChange = this.handleEventTypeChange.bind(this);
 		this.handleFilterChange = this.handleFilterChange.bind(this);
 		this.handleEventChange = this.handleEventChange.bind(this);
 		this.handleServerSideEventsChanged = this.handleServerSideEventsChanged.bind(this);
 		this.handleEventChildrenSwitch = this.handleEventChildrenSwitch.bind(this);
+		this.scrollHandler = this.scrollHandler.bind(this);
+		this.resetEventChunk = this.resetEventChunk.bind(this);
+		window.onscroll = this.scrollHandler;
+		this.handleEventChunkChange = this.handleEventChunkChange.bind(this);
+	}
+	
+	resetEventChunk() {
+		this.eventChunk = [0, config.eventTableChunkSize];
+	}
+	
+	scrollHandler() {
+		if ((window.innerHeight + window.scrollY) >= document.body.offsetHeight &&
+			this.eventChunk[1] < this.state.currentEventType.NumberOfEvents) {
+			this.eventChunk = [this.eventChunk[1] + 1, this.eventChunk[1] + config.eventTableChunkSize];
```

Comment:
How is it handled in Backend if a number is asked that doesn't exist? Because that could happen here.